lib/govuk_index/publishing_event_worker.rb

```diff
@@ -16,6 +18,9 @@ def perform(routing_key, payload)
       process_response(response) if response
     # Rescuing as we don't want to retry this class of error
     rescue ValidationError => e
+      return if e.message ==  "base_path missing from payload" &&
```

Comment:
Can we subclass ValidationError instead of checking strings here?

Reply:
Done, I actually replaced the error class as we don't currently have any other types of validation errors and I doubt we will at any point soon.